src/screens/authentication/Login/index.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react'
```

Comment:
The Login and Signup components are quite long. I'd suggest to split them out into smaller and reusable files. For example:
```javascript
    <LoginBackground>
      <LoginHeader />
      <LoginForm
        initialValues={initialValues}
        errors={errors}
        authenticating={authenticating}
        onSubmit={handleSubmit}
      />
      <NoAccountLink />
    </LoginBackground>
```